### PR TITLE
Upgrade to rancher/convoy-agent:v0.12.0

### PR DESCRIPTION
**Pull request by cjellick**

### CODE REVIEW

File: templates/convoy-nfs/5/rancher-compose.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+.catalog:
+    name: "Convoy NFS"
+    version: "v0.4.0"
```
Comment (janeczku) [inline templates/convoy-nfs/5/rancher-compose.yml:3]: The version must be `v0.5.0`

**Review by janeczku — CHANGES_REQUESTED** [review]

**Comment by janeczku** [comment]
ping @cjellick

**Comment by jabenninghoff** [comment]
can this PR be fixed and merged? The version of convoy-nfs in the catalog is still broken.

**Comment by deniseschannon** [comment]
Closing as we're moving to rancher-nfs for 1.2.0

**Comment by jabenninghoff** [comment]
Am I missing something? Isn't Convoy-NFS how you add rancher-nfs support? Frankly, issues like this have led me to give up on trying to use Rancher/RancherOS.

**Comment by deniseschannon** [comment]
@jabenninghoff We are moving to a new rancher-nfs solution in 1.2.0, which will not be using convoy anymore. 

http://docs.rancher.com/rancher/v1.2/en/rancher-services/storage-service/

**Pull request closed without merging**